pokedex/src/components/Displaying.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+const Displaying = ({ meta, isLoading }) => {
+  let displayingText = "";
+  if (meta.total !== 0) {
+    displayingText = `${meta.from} to ${meta.to} of `;
+  }
```

Comment:
This could be consolidated a little bit to something like this

`const displayingText = meta.total ? `${meta.from} to ${} of ${meta.to}` : '';

 Essentially what I'm doing is checking is `meta.total` is a truthy value, then I display the text, otherwise display a string. In your case here, if `meta.total` is equal to 0 then it'll display the right half of the ternary. This also catches if `meta.total` is equal to any other false value, or if it doesn't exist.